Build on Windows fails with "Cannot find a single highest directory"

### Steps to reproduce

1. git clone flow
2. run `mvn -C clean install -DskipTests` with CWD beginning with a lower-case drive letter.

### Expected behavior
The build succeeds

### Actual behavior
The build fails with

```
[ERROR] Failed to execute goal org.commonjava.maven.plugins:directory-maven-plugin:0.1:highest-basedir (set-root-dir-for-clean-lifecycle) on project flow-project: Cannot find a single highest directory for this project set. First two candidates directories don't share a common root. -> [Help 1]
```

### Versions:
    - Vaadin / Flow version: Flow from branch 4.0
    - Java version: 1.8
    - OS version: Windows 10

--- a/pom.xml
+++ b/pom.xml
@@ -360,7 +360,7 @@
             <plugin>
                 <groupId>org.commonjava.maven.plugins</groupId>
                 <artifactId>directory-maven-plugin</artifactId>
-                <version>0.1</version>
+                <version>0.3.1</version>
                 <executions>
                     <execution>
                         <id>set-root-dir-for-common-lifecycle</id>

